pkg/driver/controller_server.go

```diff
@@ -241,6 +241,30 @@ func (driver *Driver) CreateVolume(ctx context.Context, request *csi.CreateVolum
 		// Note: 'limit_bytes' not supported ???
 	}
 
+	// Check if volume is requested with RWX or ROX modes and intercept here
+	if driver.IsSupportedMultiNodeAccessMode(request.VolumeCapabilities) {
```

Comment:
We might want to consider that this multi-writer deployment is NOT necessary for 3par.  So if they intend to create a RWX volume, it should pass through as normal.

Reply:
got it, will add storage class parameter to filter this..